docs/tutorials/create-a-pipeline-that-blurs-faces-in-an-image.md

```diff
@@ -42,117 +42,189 @@ documentation to start them.
+                "needs": ["face-detection"],
+                "condition": "len(face-detection.result['areas']) > 0",
+                "inputs": ["pipeline.image", "face-detection.result"],
+                "service_slug": "image-blur" // Change this to your image blur service slug
             }
         ]
+    }
     ```
+    
+    !!! note
+        You can find the slug of your services by going to the FastAPI documentation of
+        the running [Core engine](../reference/core-engine.md) and use the `/services`
+        endpoint. You will find the slug of your services in the response.
+    
+        ```json hl_lines="6 11"
+            [
+                {
+                    "created_at": "2023-06-01T13:55:15.936033",
+                    "updated_at": "2023-06-01T13:55:19.831817",
+                    "name": "Face Detection",
+                    "slug": "face-detection",
+                }, {
+                    "created_at": "2023-06-01T13:55:15.936033",
+                    "updated_at": "2023-06-01T13:55:19.800560",
+                    "name": "Image Blur",
+                    "slug": "image-blur",
+                }
+            ]
+        ```
+    
+    What we just did is to create a
+    [Pipeline](../reference/core-concepts/pipeline.md) with two steps. The first
+    step is the face detection service and the second step is the image blur
+    service. The second step will only be executed if the first step detects at
+    least one face. The [Pipeline](../reference/core-concepts/pipeline.md) will take
+    an image as input and return an image as output.
+    
+    The inputs of each step are the outputs of the previous steps. The first step
+    takes the pipeline's image as input and the second step takes the
+    [Pipeline](../reference/core-concepts/pipeline.md)'s image and the result of the
+    face detection as input.
+    
+    !!! note
+        The `identifier` field of each step is the name of the step in the
+        [Pipeline](../reference/core-concepts/pipeline.md). It is used to reference the
+        step in the `needs` and `inputs` fields.
+    
+    ### Post the pipeline
+    
+    Now that we have our [Pipeline](../reference/core-concepts/pipeline.md), we can
+    post it to the [Core engine](../reference/core-engine.md). To do so, go to the
+    FastAPI documentation of the running [Core engine](../reference/core-engine.md)
+    and use the `/pipelines` endpoint to post the
+    [Pipeline](../reference/core-concepts/pipeline.md) by clicking on the
+    `Try it out` button
+    
+    Simply copy the content of the `face-blur-pipeline.json` file and paste it in
+    the `body` field of the `/pipelines` endpoint and click on the `Execute` button.
+    
+    ![Post pipeline](../assets/screenshots/post-pipeline.png)
+    
+    You should receive a `200` response with the
+    [Pipeline](../reference/core-concepts/pipeline.md) you just posted.
+
+=== "Using the Core engine interface"
```

Comment:
For one of the two title maybe this one, add (recommended) to tell the user to use this way instead of the other by default. See https://docs.swiss-ai-center.ch/how-to-guides/how-to-build-publish-and-deploy-a-service/ for an example